### DISCUSSION TITLE
How to list tag values of a CDK resource

### DISCUSSION THREAD

**Discussion by mguja-fcc** [aws/aws-cdk#24241] [General]
I can't figure out, how to list tag values of a CDK resource created with CDK Typescript? For example, multiple subnets are created and based of if a subnet's Name "subnet-outbout-d", I want to add a specific route. However, when I reference subnet.tags, I see (below). How to I get actual list of tag values ?

```
TagManager {
  tags: Map(0) {},
  priorities: Map(0) {},
  initialTagPriority: 50,
  resourceTypeName: 'AWS::EC2::Subnet',
  tagFormatter: StandardFormatter {},
  tagPropertyName: 'tags',
  renderedTags: LazyAny {
    producer: { produce: [Function: produce] },
    cache: true,
    creationStack: [ 'Execute again with CDK_DEBUG=true to capture stack traces' ],
    options: {}
  }
}
```

**Comment by indrora** [comment]
You'll need to go through the `renderedTags` property. 
https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.TagManager.html#renderedtags
